artifactory_cleanup/rules/base.py

```diff
@@ -293,21 +294,31 @@ def filter(self, artifacts: ArtifactsList) -> ArtifactsList:
                 print()
         return artifacts
 
-    def delete(self, artifact: ArtifactDict, destroy: bool) -> None:
+    def delete(self, artifact: ArtifactDict, destroy: bool, ignore_not_found: bool) -> None:
         """
         Delete the artifact
         :param artifact: artifact to remove
         :param destroy: if False - just log the action, do not actually remove the artifact
+        :param ignore_not_found: if True - do not raise an error if the artifact is not found
         """
-        path = "{repo}/{name}" if artifact["path"] == "." else "{repo}/{path}/{name}"
+
+        if artifact["path"] == ".":
+            path = "{repo}/{name}"
+        else:
+            path = "{repo}/{path}/{name}"
+
         artifact_path = path.format(**artifact)
-        artifact_path = quote(artifact_path)
-        artifact_size = artifact.get("size", 0) or 0
+        artifact_size = artifact.get("size", 0)
```

Comment:
`artifact_path = quote(artifact_path)` has been removed - is it on purpose? 
```suggestion
        artifact_path = quote(artifact_path)
        artifact_size = artifact.get("size", 0)
```

Reply:
Also I'm not sure why there was added `or`, but may some of dict can be WITH `size` set to None or empty string? In that case the code would give the different result. 
I'd keep it as it was before, if you don't mind
```suggestion
        artifact_path = quote(artifact_path)
        artifact_size = artifact.get("size", 0) or 0
```